src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpTransportContext.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Security.Authentication.ExtendedProtection;
+
+using SafeInternetHandle = Interop.WinHttp.SafeInternetHandle;
+
+namespace System.Net.Http
+{
+    internal class WinHttpTransportContext : TransportContext
+    {
+        private WinHttpChannelBinding _channelBinding = null;
```

Comment:
@bartonjs I don't know if CodeFormatter removes them or not.  I assume you think it is redundant because doing a 'new' of any object will initialize all fields to null / zero.

From a coding guidelines viewpoint and developer understanding, we found it valuable to always initialize the fields to a known value.   We also assume that the compiler optimizer will do the right thing there.